woocommerce/woocommerce-gutenberg-products-block
# MVP: Attribute filter block powered interactivity API

## What

Partial fixes #10812 

This PR adds the initial version of the new attribute filter block powered by Interactivity API. This MVP supports:
- Display product count.
- Filter conditions.
- Display style: Dropdown/List.
- Multiple/single selections ( Dropdown hasn't supported multiple selections yet).
## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Add the `Collection Filters` block to the `Product Collection` block.
2. Add the `Collection Attribute Filter` block to the `Collection Filters` block.
3. If the site doesn't have any attributes, see the warning informing the merchant.
4. If the site has attributes, see the attribute selector.
5. Select the attribute > Done.
6. See the attribute list loading.
7. Try sidebar settings, see the editor preview react with the change.
8. Save and see the setting applied on the frontend.
9. Ensure other new filter blocks work together.


## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Add suggested changelog entry here.
